network/monuments.csv

```diff
@@ -116,6 +116,7 @@ OHIN,,Forced Centering,Wyatt/Agnew Drilled-Braced,-1.6,Stainless Steel Rods,10,2
 OKOH,,Forced Centering,Wyatt/Agnew Drilled-Braced,-1.56,Stainless Steel Rods,10,2009-11-16T00:00:00Z,9999-01-01T00:00:00Z,Greywacke,
 OPTK,,Forced Centering,Wyatt/Agnew Drilled-Braced,-1.46,Stainless Steel Rods,10,2009-08-20T00:00:00Z,9999-01-01T00:00:00Z,,
 OROA,,Forced Centering,Wyatt/Agnew Drilled-Braced,0,Stainless Steel Rods,10,2010-11-08T00:00:00Z,9999-01-01T00:00:00Z,Sandstone,
+ORRG,,Forced Centering,Wyatt/Agnew Drilled-Braced,-1.5,Stainless Steel Rods,10,2024-12-05T00:00:00Z,9999-01-01T00:00:00Z,,
```

Comment:
Hey @Evan! Could you please put here more info from the geological map?

Reply:
Done!